src/components/base/JCheckbox/JCheckbox.js

```diff
@@ -43,13 +53,14 @@ class JCheckbox extends PureComponent<Props> {
           <span className='flag' />
           <span className='label'>
             <JText
-              color='gray'
+              color={color}
               size='normal'
               value={label}
-              weight='bold'
+              weight={isRegular ? 'regular' : 'bold'}
```

Comment:
just pass `null`, or make this prop required, because now there are two ways to define "regular" font weight